frontend/src/components/Login.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react'
+
+const SIGNUP_URL = 'https://project-auth-technigo.herokuapp.com/users' 
+const LOGIN_URL = 'https://project-auth-technigo.herokuapp.com/sessions'
+const SECRETS_URL ='https://project-auth-technigo.herokuapp.com/secrets'
+
+
+export const Login = () => {
+  const [email, setEmail] = useState('')
+  const [password, setPassword] = useState('')
+  const [secrets, setSecrets] = useState('')
+  const [status, setStatus] = useState('')
+
+  const validateEmptyMessage = message => message.replace(/\s/g, "").length === 0
+  const validateEmail = (email) => {
+    return /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(email)
+  }
+
+  const handleSignup = (event) => {
+    event.preventDefault()
+  
+    fetch(SIGNUP_URL, {
+      method: 'POST',
+      body: JSON.stringify({ email, password }),
+      headers: { 'Content-Type': 'application/json' }
+    })
+      .then(res => res.json())
+      .then(json => {
+        setStatus(json.message)
+      })
+      .catch(err => console.log('error:', err))
+  }
+
+  const handleLogin = (event) => {
+    event.preventDefault()
+
+    fetch(LOGIN_URL, {
+      method: 'POST',
+      body: JSON.stringify({ email, password }),
+      headers: { 'Content-Type': 'application/json' }
+    })
+      .then(res => res.json())
+      .then(json => {
+        handleSecrets(json.accessToken)
+        setStatus(json.message)
+      })
+      .catch(err => console.log('error:', err))
+  }
+
+  const handleSecrets = (accessToken) => {
+    fetch(SECRETS_URL, {
+      method: 'GET',
+      headers: { Authorization: accessToken }
+    })
+      .then(res => res.json())
+      .then(json => {
+        setSecrets(json.secret)
+      })
+      .catch(err => {console.log('error:', err)
+    })
+  }
+
+  return (
+    <>
+      {!secrets ? (
+        <>
+          <section className="login">
+            <h1>Log in / Sign up</h1>
+            <form>
+              <input
+                type="email"
+                minLength="8"
+                placeholder="Email"
+                required
+                value={email}
+                onChange={event => setEmail(event.target.value)}
+              />
```

Comment:
Consider to add maxLength="12" since the button is disabled if you write more than 12. It goes for the password as well.